Package.swift

```diff
@@ -334,31 +334,5 @@ var streamChatUIFilesExcluded: [String] { [
-    "Common Views/CurrentChatUserAvatarView_Tests.swift",
-    "Common Views/ChatChannelCreateNewButton_Tests.swift",
-    "Common Views/__Snapshots__/ChatChannelCreateNewButton_Tests/test_isDisabled.default-light.png",
-    "Common Views/__Snapshots__/ChatChannelCreateNewButton_Tests/test_customizationUsingUIConfig.default-dark.png",
-    "Common Views/__Snapshots__/ChatChannelCreateNewButton_Tests/test_defaultAppearance.default-light.png",
-    "Common Views/__Snapshots__/ChatChannelCreateNewButton_Tests/test_customizationUsingUIConfig.default-light.png",
-    "Common Views/__Snapshots__/ChatChannelCreateNewButton_Tests/test_customizationUsingSubclassingHook.default-light.png",
-    "Common Views/__Snapshots__/ChatChannelCreateNewButton_Tests/test_isDisabled.default-dark.png",
-    "Common Views/__Snapshots__/ChatChannelCreateNewButton_Tests/test_isHighlighted.default-light.png",
-    "Common Views/__Snapshots__/ChatChannelCreateNewButton_Tests/test_defaultAppearance.default-dark.png",
-    "Common Views/__Snapshots__/ChatChannelCreateNewButton_Tests/test_customizationUsingAppearanceHook.default-light.png",
-    "Common Views/__Snapshots__/ChatChannelCreateNewButton_Tests/test_customizationUsingSubclassingHook.default-dark.png",
-    "Common Views/__Snapshots__/ChatChannelCreateNewButton_Tests/test_customizationUsingAppearanceHook.default-dark.png",
-    "Common Views/__Snapshots__/ChatChannelCreateNewButton_Tests/test_isHighlighted.default-dark.png",
-    "Common Views/__Snapshots__/CurrentChatUserAvatarView_Tests/test_defaultAppearance.default-light.png",
-    "Common Views/__Snapshots__/CurrentChatUserAvatarView_Tests/test_customizationUsingAppearanceHook.default-light-empty.png",
-    "Common Views/__Snapshots__/CurrentChatUserAvatarView_Tests/test_customizationUsingSubclassingHook.default-light.png",
-    "Common Views/__Snapshots__/CurrentChatUserAvatarView_Tests/test_defaultAppearance.default-dark.png",
-    "Common Views/__Snapshots__/CurrentChatUserAvatarView_Tests/test_emptyAppearance.default-light.png",
-    "Common Views/__Snapshots__/CurrentChatUserAvatarView_Tests/test_emptyAppearance.default-dark.png",
-    "Common Views/__Snapshots__/CurrentChatUserAvatarView_Tests/test_customizationUsingAppearanceHook.default-dark-empty.png",
-    "Common Views/__Snapshots__/CurrentChatUserAvatarView_Tests/test_customizationUsingSubclassingHook.default-light-empty.png",
-    "Common Views/__Snapshots__/CurrentChatUserAvatarView_Tests/test_customizationUsingSubclassingHook.default-dark-empty.png",
-    "Common Views/__Snapshots__/CurrentChatUserAvatarView_Tests/test_customizationUsingAppearanceHook.default-light.png",
-    "Common Views/__Snapshots__/CurrentChatUserAvatarView_Tests/test_customizationUsingSubclassingHook.default-dark.png",
-    "Common Views/__Snapshots__/CurrentChatUserAvatarView_Tests/test_customizationUsingAppearanceHook.default-dark.png"
```

Comment:
This is actually right, but will be fixed on main so after rebase this disappears. :)